docs/api/options.md

```diff
@@ -292,29 +292,21 @@ When `sync` is `false`, the Vue component is rendered asynchronously.
 
 ## Other options
 
-When the options for `mount` and `shallowMount` contain the options other than the mounting options, the component options are overwritten with those using [extends](https://vuejs.org/v2/api/#extends).
+When the options for `mount` and `shallowMount` contain the options other than the mounting options, the options are added used when the Vue instance is initialized.
 
 ```js
 const Component = {
-  template: '<div>{{ foo() }}{{ bar() }}{{ baz() }}</div>',
-  methods: {
-    foo () {
-      return 'a'
-    },
-    bar () {
-      return 'b'
-    }
-  }
+  template: `
```

Comment:
This example could be improved to make context more clear. 
`expect(wrapper.text()).toBe('aBC')` would fail?